src/storage/storage_info.cpp

```diff
@@ -2,7 +2,7 @@
 
 namespace duckdb {
 
-const uint64_t VERSION_NUMBER = 64;
+const uint64_t VERSION_NUMBER = 65;
```

Comment:
What is the reason for increasing the storage version?

I'm always a little fuzzy on this to be honest, but I believe VERSION_NUMBER should only be changed when we break the storage (?)
Which is not the case here, we've only had to update the `storage_version.db` because the  results of `generate_storage_version.sql` have changed.

Reply:
That indeed makes more sense. I followed the instructions at the top of the storage version test blindly.